ruby-bugs.csv

```diff
@@ -3,11 +3,11 @@
 4173,test,TestProcess#test_wait_and_sigchild が、たまに失敗する
 4352,ruby3 core,"[patch] Fix eval(s, b) backtrace; make eval(s, b) consistent with eval(s)"
 4405,win thread ext,WIN32OLE &amp; Threads incompatible
-4434,thread,make test-all &quot;-j10000&quot; のように大きな並列数を与えると異常終了
+4434,thread maybenotbug,make test-all &quot;-j10000&quot; のように大きな並列数を与えると異常終了
```

Comment:
This could be a real issue, but I highly doubt someone ever needs this amount of threads/processes in Ruby (though you could run into it when the resource limit is set to a very low limit).